File: Swift_Code/iOS -Objective-C调用Swift代码/CallSwift/ViewController.m
//
//  ViewController.m
//  CallSwift
//
//  Created by tianXin on 2017/2/22.
//  Copyright © 2017年 tianXin. All rights reserved.
//

#import "ViewController.h"
#import "CallSwift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    BYSAppHelp *help = [[BYSAppHelp alloc] init];
    [help callSwift];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Swift_Code/iOS-Swift与Objective-C混合编程/iOS-Swift与Objective-C混合编程/ViewController.swift
//
//  ViewController.swift
//  iOS-Swift与Objective-C混合编程
//
//  Created by tianXin on 2017/2/22.
//  Copyright © 2017年 tianXin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let help = BYSAppHelp()
        help.getAppFont()
        
        print(help)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

